stdlib/threading.pyi

```diff
@@ -109,6 +109,10 @@ class Lock:
     def acquire(self, blocking: bool = ..., timeout: float = ...) -> bool: ...
     def release(self) -> None: ...
     def locked(self) -> bool: ...
+    if sys.version_info >= (3, 13):
+        def acquire_lock(self, blocking: bool = ..., timeout: float = ...) -> bool: ...  # undocumented
+        def release_lock(self) -> None: ...  # undocumented
+        def locked_lock(self) -> bool: ...  # undocumented
```

Comment:
I think these always existed, we just see them only on 3.13+ because Lock() used to be a function at runtime so stubtest couldn't find these methods.

```
% ~/.pyenv/versions/3.8.16/bin/python -c 'import threading; print(threading.Lock().acquire_lock)'
<built-in method acquire_lock of _thread.lock object at 0x100db1c30>
```

```suggestion
    def acquire_lock(self, blocking: bool = ..., timeout: float = ...) -> bool: ...  # undocumented
    def release_lock(self) -> None: ...  # undocumented
    def locked_lock(self) -> bool: ...  # undocumented
```

Reply:
Ah, right. I'll remove my comment on the CPython issue too.